src/main/java/com/epam/aidial/core/config/Deployment.java

```diff
@@ -26,4 +26,5 @@ public abstract class Deployment {
     private Features features;
     private List<String> inputAttachmentTypes;
     private Integer maxInputAttachments;
+    private String apiKey;
```

Comment:
Will it be included in listing response?

Reply:
no, it's not by security reasons.